validation/src/test/scala/hmda/validation/rules/lar/validity/V220Spec.scala

```diff
@@ -13,4 +15,18 @@ class V220Spec extends PropSpec with PropertyChecks with MustMatchers with LarGe
       }
     }
   }
+
+  val badLoanTypeGen: Gen[Int] = {
+    val belowRange = Gen.choose(Integer.MIN_VALUE, 0)
+    val aboveRange = Gen.choose(5, Integer.MAX_VALUE)
+    Gen.oneOf(belowRange, aboveRange)
+  }
+
+  property("Loan Type other than 1,2,3,4 is invalid") {
+    forAll(larGen, badLoanTypeGen) { (lar: LoanApplicationRegister, x: Int) =>
+      val invalidLoan: Loan = lar.loan.copy(loanType = x)
+      V220(invalidLoan) mustBe Failure("is not contained in valid values domain")
```

Comment:
One of the things that is being discussed as a potential change is the error messages displayed to the user